ogc-sys/src/ogc.rs

```diff
@@ -75,11 +75,10 @@ impl<T> ::core::cmp::PartialEq for __BindgenUnionField<T> {
 impl<T> ::core::cmp::Eq for __BindgenUnionField<T> {}
 pub const __NEWLIB_H__: u32 = 1;
 pub const _NEWLIB_VERSION_H__: u32 = 1;
-pub const _NEWLIB_VERSION: &'static [u8; 6usize] = b"4.1.0\0";
-pub const __NEWLIB__: u32 = 4;
-pub const __NEWLIB_MINOR__: u32 = 1;
+pub const _NEWLIB_VERSION: &'static [u8; 6usize] = b"3.3.0\0";
```

Comment:
What version of ``libogc`` do you currently have installed? It is odd for the newlib version to decrease unless they reverted changes.

Reply:
I'm running `Arch Linux` so I'd expect to have the latest version of `libogc`. `ogc.rs` gets manually built with `ProfElements` work though, so I think it's better to move to that instead.

Reply:
Sounds good, once merged they can be updated to the new method.